xds/type/v3/cel.proto

```diff
@@ -47,6 +47,11 @@ message CelExpression {
   //
   // If set, takes precedence over ``cel_expr_parsed``.
   cel.expr.CheckedExpr cel_expr_checked = 4;
+
+  // Unparsed expression in string form.
+  //
+  // If set, takes precedence over ``cel_expr_parsed`` and ``cel_expr_checked``.
```

Comment:
Please add a description or link to what is a valid string value that is expected here.

Reply:
An example and the spec link is added.

Reply:
Thanks!
I'm going to approve this for now, but it's not clear to me where is "request.headers" defined? Is there a "response.body"?
I would assume this is all written somewhere (doc), and there should be a link here to that doc.

Reply:
Most CEL attributes are host-dependent. That's say, the example requires the host to support `request.headers` attributes. And you can see all attributes supported by envoy at here: https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/advanced/attributes

Reply:
No sure because xds repo is also be used by other projects. So, I only put an example and spec link here.